storage/response_cache.go

```diff
@@ -151,10 +151,21 @@ func (rc *ResponseCache) CopyInto(e engine.Engine, destRaftID int64) error {
 		// write it to the corresponding key in the new cache.
 		cmdID, err := rc.decodeResponseCacheKey(kv.Key)
 		if err != nil {
-			return false, util.Errorf("could not decode a response cache key %q: %s", kv.Key, err)
+			return false, util.Errorf("could not decode a response cache key %s: %s",
```

Comment:
Yeah, I'll take a look at switching this to MVCCScan. Definitely ugly the way it is now.